2020-02-Spring/0_arrays_matrix/README.md

```diff
@@ -50,19 +50,25 @@ Go to [Solution](#s1)   [Top](#top)
 
 ### 2. PROBLEM 2 TODO :bug:
 
-Source: TODO :bug:
+Source: https://leetcode.com/problems/maximum-subarray/
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Given an integer array nums, find the contiguous subarray (containing at least one number) which has the largest sum and return its sum.
```

Comment:
I'm adding a small change here, but the explanation needs a little more detail. Things like what is considered correct input, how to format output, whether or not correct input is guaranteed, etc.

```suggestion
Given an array of integers, find the contiguous subarray (containing at least one number) which has the largest sum and return the sum.
```